packages/react/src/components/Button/index.spec.tsx

```diff
@@ -19,6 +19,17 @@ describe("<Button/>", () => {
     expect(button.classList.contains("c__button--with-icon--left")).toBe(true);
   });
 
+  it("renders with modifier class --icon-only when only icon is defined", () => {
+    render(<Button icon={<div>Icon</div>} />);
+    const button = screen.getByRole("button");
+    expect(button.classList.contains("c__button")).toBe(true);
```

Comment:
```suggestion
    expect(button.classList).toContain("c__button");
```
?

Reply:
I updated all the test file to match this neater pattern.